### ISSUE TITLE
If bootkube times out on first boot, subsequent tries fails

### ISSUE THREAD

**Issue by salkin** [talos-systems/talos#2969]
### Description
If bootkube for any reason does not complete on first try any subsequent boot of same node fails

### Logs

``` 
bash-5.0# talosctl --talosconfig talos.yaml logs bootkube
192.168.253.101: Writing asset: /etc/kubernetes/assets/manifests/kube-scheduler-disruption.yaml
192.168.253.101: Writing asset: /etc/kubernetes/assets/manifests/kube-controller-manager-disruption.yaml
192.168.253.101: Writing asset: /etc/kubernetes/assets/manifests/coredns-cluster-role-binding.yaml
192.168.253.101: Writing asset: /etc/kubernetes/assets/manifests/coredns-cluster-role.yaml
192.168.253.101: Writing asset: /etc/kubernetes/assets/manifests/coredns-deployment.yaml
192.168.253.101: Writing asset: /etc/kubernetes/assets/manifests/coredns-service-account.yaml
192.168.253.101: Writing asset: /etc/kubernetes/assets/manifests/pod-checkpointer.yaml
192.168.253.101: Writing asset: /etc/kubernetes/assets/manifests/pod-checkpointer-sa.yaml
192.168.253.101: Writing asset: /etc/kubernetes/assets/manifests/pod-checkpointer-role.yaml
192.168.253.101: Writing asset: /etc/kubernetes/assets/manifests/pod-checkpointer-role-binding.yaml
192.168.253.101: Writing asset: /etc/kubernetes/assets/manifests/pod-checkpointer-cluster-role.yaml
192.168.253.101: Writing asset: /etc/kubernetes/assets/manifests/pod-checkpointer-cluster-role-binding.yaml
192.168.253.101: Writing asset: /etc/kubernetes/assets/manifests/csr-approver-role-binding.yaml
192.168.253.101: Writing asset: /etc/kubernetes/assets/manifests/csr-bootstrap-role-binding.yaml
192.168.253.101: Writing asset: /etc/kubernetes/assets/manifests/csr-renewal-role-binding.yaml
192.168.253.101: Writing asset: /etc/kubernetes/assets/manifests/kube-system-rbac-role-binding.yaml
192.168.253.101: Writing asset: /etc/kubernetes/assets/manifests/kube-scheduler.yaml
192.168.253.101: Writing asset: /etc/kubernetes/assets/manifests/kube-controller-manager.yaml
192.168.253.101: Writing asset: /etc/kubernetes/assets/manifests/kube-controller-manager-service-account.yaml
192.168.253.101: Writing asset: /etc/kubernetes/assets/manifests/kube-controller-manager-role-binding.yaml
192.168.253.101: Writing asset: /etc/kubernetes/assets/manifests/kube-apiserver.yaml
192.168.253.101: Writing asset: /etc/kubernetes/assets/manifests/kube-proxy.yaml
192.168.253.101: Writing asset: /etc/kubernetes/assets/manifests/kube-proxy-sa.yaml
192.168.253.101: Writing asset: /etc/kubernetes/assets/manifests/kube-proxy-role-binding.yaml
192.168.253.101: Writing asset: /etc/kubernetes/assets/manifests/coredns-config.yaml
192.168.253.101: Writing asset: /etc/kubernetes/assets/manifests/coredns-service.yaml
192.168.253.101: Writing asset: /etc/kubernetes/assets/bootstrap-manifests/bootstrap-apiserver.yaml
192.168.253.101: Writing asset: /etc/kubernetes/assets/bootstrap-manifests/bootstrap-controller-manager.yaml
192.168.253.101: Writing asset: /etc/kubernetes/assets/bootstrap-manifests/bootstrap-scheduler.yaml
192.168.253.101: Writing asset: /etc/kubernetes/assets/tls/encryptionconfig.yaml
192.168.253.101: Writing asset: /etc/kubernetes/assets/tls/auditpolicy.yaml
192.168.253.101: Writing asset: /etc/kubernetes/assets/manifests/psp.yaml
192.168.253.101: Writing asset: /etc/kubernetes/assets/tls/ca.key
192.168.253.101: Writing asset: /etc/kubernetes/assets/tls/ca.crt
192.168.253.101: Writing asset: /etc/kubernetes/assets/tls/apiserver.key
192.168.253.101: Writing asset: /etc/kubernetes/assets/tls/apiserver.crt
192.168.253.101: Writing asset: /etc/kubernetes/assets/tls/service-account.key
192.168.253.101: Writing asset: /etc/kubernetes/assets/tls/front-proxy-ca.crt
192.168.253.101: Writing asset: /etc/kubernetes/assets/tls/front-proxy-client.crt
192.168.253.101: Writing asset: /etc/kubernetes/assets/tls/front-proxy-client.key
192.168.253.101: Writing asset: /etc/kubernetes/assets/tls/apiserver-kubelet-client.key
192.168.253.101: Writing asset: /etc/kubernetes/assets/tls/apiserver-kubelet-client.crt
192.168.253.101: Writing asset: /etc/kubernetes/assets/tls/service-account.pub
192.168.253.101: Writing asset: /etc/kubernetes/assets/tls/admin.key
192.168.253.101: Writing asset: /etc/kubernetes/assets/tls/admin.crt
192.168.253.101: Writing asset: /etc/kubernetes/assets/tls/etcd-client-ca.crt
192.168.253.101: Writing asset: /etc/kubernetes/assets/tls/etcd-client.key
192.168.253.101: Writing asset: /etc/kubernetes/assets/tls/etcd-client.crt
192.168.253.101: Writing asset: /etc/kubernetes/assets/auth/kubeconfig
192.168.253.101: Writing asset: /etc/kubernetes/assets/manifests/kubeconfig-in-cluster.yaml
192.168.253.101: Writing asset: /etc/kubernetes/assets/auth/kubeconfig-kubelet
192.168.253.101: Writing asset: /etc/kubernetes/assets/manifests/kubelet-bootstrap-token.yaml
192.168.253.101: Writing asset: /etc/kubernetes/assets/manifests/kube-apiserver-secret.yaml
192.168.253.101: Writing asset: /etc/kubernetes/assets/manifests/kube-controller-manager-secret.yaml
192.168.253.101: Starting temporary bootstrap control plane...
192.168.253.101: Error: open /etc/kubernetes/manifests/bootstrap-apiserver.yaml: file exists
192.168.253.101: Tearing down temporary bootstrap control plane...
192.168.253.101: I1221 08:46:40.867857       1 log.go:19] 2020/12/21 08:46:40 bootkube failed: open /etc/kubernetes/manifests/bootstrap-apiserver.yaml: file exists
```

### Environment

- Talos version: [`talosctl version --nodes <problematic nodes>`]
``` 
bash-5.0# talosctl --talosconfig talos.yaml version
Client:
        Tag:         v0.7.1
        SHA:         333bc8fd
        Built:
        Go version:  go1.15.5
        OS/Arch:     linux/amd64

Server:
        NODE:        192.168.253.101
        Tag:         v0.7.1
        SHA:         333bc8fd
        Built:
        Go version:  go1.15.5
        OS/Arch:     linux/amd64
``` 

- Platform: Qemu or baremetal

**Comment by salkin** [comment]
after few reboots it seems to start again so there seems to be something with the first boot failure timeout that does not clean up?

**Comment by smira** [comment]
@salkin it was fixed in 0.8.0-beta.1

**Comment by smira** [comment]
PR #2962
